testing/test_junitxml.py

```diff
@@ -619,6 +620,20 @@ def test_func(char):
     node.assert_attr(name="test_func[#x00]")
 
 
+def test_double_colon_split_issue469(testdir):
```

Comment:
Yeah I originally wrote one, but changed by mind!

Added in https://github.com/pytest-dev/pytest/pull/1432